equipment.py

```diff
@@ -0,0 +1,101 @@
+import datajoint as dj
+
+schema = dj.schema(dj.config['names.%s' % __name__], locals())
+
+
+@schema
+class LabLocation(dj.Manual):
+    # <class 'equipment.models.LabLocation'>
+    definition = """
+    lab_location:		int             # lab location
+    ---
+    name:                       varchar(255)    # name
+    """
+
+
+@schema
+class Supplier(dj.Manual):
+    # <class 'equipment.models.Supplier'>
+    definition = """
+    supplier_name:		int             # supplier name
+    ---
+    description:		varchar(1024)	# description
+    """
+
+
+@schema
+class VirusBatch(dj.Manual):
+    # <class 'equipment.models.VirusBatch'>
+    definition = """
+    -> Supplier
+    batch_id:			int             # batch id
+    ---
+    description:		varchar(255)	# description
+    virus_type:			varchar(255)	# virus type
+    date_time_made:		datetime	# date time made
+    nominal_titer:		float		# nominal titer
+    """
+
+
+@schema
+class EquipmentModel(dj.Manual):
+    # <class 'equipment.models.EquipmentModel'>
+    definition = """
+    -> Supplier
+    model_id:			int             # model id
+    ---
+    model_name:			varchar(255)	# model name
+    description:		varchar(255)	# description
+    """
+
+
+@schema
+class Appliance(dj.Manual):
+    # <class 'equipment.models.Appliance'>
+    # TODO: Should appliances be a lookup?
+    # ... assuming Alyx' separate classes vs field is reasoned.
+    definition = """
+    -> LabLocation
+    -> EquipmentModel
```

Comment:
To me, it appears that `EquipmentModel` specifies exactly what kind of appliance this is already and thus use of specific part tables feel a bit redundant. If anything, breakdown into specific object category ought to be done at the level of `EquipmentModel` (i.e. `EquipmentModel` can have part tables like `WeighingScale` etc) instead.

Reply:
perhaps merging EquipmentModel fields into Appliance and zapping EquipmentModel makes more sense?